apis/apps/pub/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
I'm not sure how you add the duplicated line in generated files... maybe have to remove them.

Reply:
I run `make test` and auto generate this line, don't know why 🤔.